examples/sample/release-with-crds.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.crossplane.io/v1beta1
+kind: Release
+metadata:
+  name: argo-cd-example
+spec:
+  forProvider:
+    chart:
+      name: argo-cd
+      repository: https://charts.bitnami.com/bitnami
+      version: 2.0.12
+    namespace: argo-cd
+    skipCRDs: true
```

Comment:
Instead of new example yaml, please add the optional config as commented out [like this](https://github.com/crossplane-contrib/provider-helm/blob/e49c3e791f5c7bd95d50ab0762297daeb084c459/examples/sample/release.yaml#L17).

Reply:
I was considering that option from the beginning. But the problem with `wordpress` chart is that it doesn't install CRDs. As result, `spec.forProvider.skipCRDs` won't have any effect on deployment, and the example won't be true to life.
That's why I've added a separate example with `argo-cd` chart. 
But sure, I can amend.

Reply:
Agree, amended. Using argo-cd helm as the original release example would require extra efforts on mapping all parameter examples to real-world argo-cd deployment cases, might be an extra enhancement outside of this PR :)